drivers/CoreIMU/include/CoreAccelerometer.h

```diff
@@ -0,0 +1,25 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "CoreLSM6DSOX.h"
+#include "interface/Accelerometer.h"
+
+namespace leka {
+
+using mg_t = float;
+
+class CoreAccelerometer : public interface::Accelerometer
```

Comment:
**Note for myself.** CoreAccelerometer specific to LSM6DSOX

Reply:
Somehow, `CoreLSM6DSOX` should provide any tools for `CoreAccelerometer` and `CoreGyroscope` without knowing any data nature.

A small part of `CoreLSM6DSOX` leads me in a wrong way with an initialization of accelerometer and gyroscope specific configuration (rate and range).

Same for the power mode setting with the use of accelerometer (xl) and not the gyroscope.

---

Just to be sure, @ladislas @HPezz do you agree with the following

- CoreLSM6DSOX is a bridge to ST's driver without specification of data (notion of rate, notion of range in g or dps)
  - General configuration of component (turn off, disable i3c, power mode (?), ...)
- CoreAccelerometer fetch data from CoreLSM6DSOX and output it to be consider as accelerometer data type
  - Apart of getting data and format it, the class allows to configure the component, as example rate and range
- CoreGyroscope same as above for gyroscope